dockers/base-ipu/Dockerfile

```diff
@@ -56,13 +57,12 @@ ENV \
 
 COPY environment.yml environment.yml
 
-RUN conda create -y --name $CONDA_ENV python=${PYTHON_VERSION} pytorch=${PYTORCH_VERSION} cudatoolkit=${CUDA_VERSION} -c pytorch && \
-    conda init bash && \
+RUN conda init bash && \
```

Comment:
one general question: why don't we start with a miniconda image in the first place?

Reply:
No idea. I can refactor the image to it, but that's not work for this PR, in this one I tried to just improve building times while keeping the image as similar to the original as possible

Reply:
agreed, but this would further reduce building time on our end :)

Reply:
Resolving to unblock master. We can consider further reductions separately